python/F3DPythonBindings.cxx

```diff
@@ -116,7 +122,8 @@ PYBIND11_MODULE(pyf3d, module)
     .def_property("content", getImageBytes, setImageBytes)
     .def("compare", &f3d::image::compare)
     .def(
-      "save", &f3d::image::save, py::arg("path"), py::arg("format") = f3d::image::SaveFormat::PNG);
+      "save", &f3d::image::save, py::arg("path"), py::arg("format") = f3d::image::SaveFormat::PNG)
+    .def("save_buffer", getFileBytes, py::arg("format") = f3d::image::SaveFormat::PNG);
```

Comment:
```suggestion
    .def("save_buffer", getFileBytes, py::arg("format") = f3d::image::SaveFormat::PNG)
    .def("_repr_png_",
      [&getFileBytes](const f3d::image& img)
      { return getFileBytes(img, f3d::image::SaveFormat::PNG); });
```
implementing `_repr_png_` in the class is an alternative to registering the `"image/png"` formatter from `__init__.py`
- pros: no dynamic `try: import ... except ImportError: pass` funny business in the module
- cons: extra seemingly random method in the class? not sure if it's a big deal actually

Reply:
Interesting! I think I prefer this version as it does not rely directly on IPython module installed. What's your preference @snoyer?

Reply:
The naming convention is ugly but it seems to be standard so I'm leaning towards this one too.
If it's good enough for Pillow it's good enough for me: https://github.com/python-pillow/Pillow/blob/main/src/PIL/Image.py#L658-L663